src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
```

Comment:
The function name 'calculateRentalCost' is descriptive enough, but the parameter 'days' could be more descriptive by naming it 'numberOfDays' to clarify its purpose.